contributors/join-us/recruiting.md

```diff
@@ -19,57 +19,127 @@ If you're considering a role at Mattermost, here are the **Top 10 Reasons to wor
+* Teams will also plan meetups, with locations determined by the Team Lead.
+
+**Additional resources**
+
+* [Mattermost Blog](https://mattermost.com/blog/)
+* [Mattermost Leadership Principles](https://handbook.mattermost.com/company/about-mattermost#leadership-principles)
+
+### Interview Process
+
+We can hire from many countries and regions across the globe. While in-office interviews typically go through 1 - 3 interview rounds, our candidates and interviewers may be located in various timezones and therefore each interview is scheduled separately.
+
+## Interview Stages
 
 1. **Recruiter Screen:** This person is a candidate's main point of contact during the interview process at Mattermost. For some roles, Hiring Managers may incorporate a recruiter screen within their first interview with a candidate.
-2. **Hiring Manager, Technical, and Team Interviews:** This portion typically consists of 3 - 4 interviewers. In addition to the Hiring Manager, candidates can expect to interview with team and cross-team members who will work closely with, or similarly to, the individual in this role.
+2. **Hiring Manager, Technical, and Team Interviews:** This portion typically consists of 3-4 interviewers. In addition to the Hiring Manager, candidates can expect to interview with team and cross-team members who will work closely with, or similarly to, the individual in this role.
 3. **Interview Project:** All take-home projects will list the expected amount of time expected to complete. We really appreciate all of the time spent completing a take-home project. As a 'thank you,' candidates will be invited to select a gift option upon completion of the project.
 4. **Final Interviews:** This typically includes the Hiring Team's [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) and one of our co-founders. **Note:** For Director and above interviews, both Mattermost co-founders will interview candidates.
 5. **Final Decision:** A member of the Hiring Team will reach out to you to let you know our decision.
 
 We hope that you have a great experience interviewing at Mattermost and welcome any feedback you may have. We may invite candidates to share feedback about our interview process so we may continually improve the process and experience.
 
-## Open Roles
+## Reference Checks and Offer Stage
 
-Mattermost's openings can be found [here](https://mattermost.com/careers/).
+Candidates will receive a verbal offer to join Mattermost. Upon verbal acceptance, the hiring team will send the contract paperwork to the new candidate. Offer letters will be signed internally first, with the [CAO](The stages are as follows: [CAO to initial](https://handbook.mattermost.com/operations/operations/company-processes/company-agreements#who-are-contract-accountability-owners-caos) as the first signer. The to-be-new staff member will be the third and final signer.
```

Comment:
This line is incorrectly formatted, but I wasn't sure what is the proper form. Let me know if you need help - feel free to post the intended text, and Justine or I can help with the Markdown formatting.